content/concepts/kata/beta-process.md

```diff
@@ -5,98 +5,46 @@ sidebar: explanation-kata
-
-<table>
-<thead>
-<tr>
-<th>Level</th>
-<th>Satisfaction %</th>
-<th colspan="4">Required Votes by # of Approved Katas</th>
-</tr>
-<tr>
-<th></th>
-<th></th>
-<th>&lt;10</th>
-<th>&lt;20</th>
-<th>&lt;30</th>
-<th>30+</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>White</td>
-<td>80+</td>
-<td>12</td>
-<td>10</td>
-<td>8</td>
-<td>6</td>
-</tr>
-<tr>
-<td>Yellow</td>
-<td>80+</td>
-<td>10</td>
-<td>8</td>
-<td>7</td>
-<td>5</td>
-</tr>
-<tr>
-<td>Blue</td>
-<td>75+</td>
-<td>8</td>
-<td>6</td>
-<td>5</td>
-<td>4</td>
-</tr>
-<tr>
-<td>Purple</td>
-<td>70+</td>
-<td>5</td>
-<td>4</td>
-<td>3</td>
-<td>3</td>
-</tr>
-</tbody>
-</table>
-
-## Auto-Retire
-
-Low quality kata are automatically retired. Current rules are:
-
-1. If 4 or more votes and satisfaction percent is `0` (4+ consecutive "Not Satisfied")
-2. If `n` or more votes and satisfaction percent is less than or equal to the half of the required satisfaction (see above). `n` is the same as the number of votes needed to be approved (see above).
-
-This feature needs improvement and any feedback is appreciated. Please comment on [#1672](https://github.com/Codewars/codewars.com/issues/1672).
+Each kata is crafted for and by the community. Kata are created by other users rewarded with kata authoring privilege. When a kata is published by an author (or Sensei), it will go into a beta process where the community can help test and provide quality assurance feedback and cast a vote for its difficulty ranking. The feedback during this process is vital to making sure all kata are of the highest quality.
+
+## Reward for Reviewers
+
+You earn an _additional_ 2 honor for every Beta Kata you solve, in order to reward you for taking the risk of solving a Kata with potential issues. Then, leaving a satisfaction vote and rank assessment on said Beta Kata rewards you with 1 _extra_ honor _each_, in order to reward you for taking the time to leave valuable feedback on said Kata. Finally, if the Kata is approved, you will also receive the honor and rank progress for the official Kata. Therefore, it is in everyone's interest that you leave your satisfaction vote and rank assessment on every Beta Kata you have completed.
+
+## Steps of Beta Process
+
+Some of the steps are available only after earnig the required privilege, but every user can help reviewing a kata in some way.
+
+### Attempting the Kata
+
+Training on a kata and solving it is the first step to verify its quality, but also a very effective one. Reading through description, users can check if it's clear enough, or something needs to be reworded or explained in a clearer way. When tests are run, reviewers can verify if they are sufficient, or whether assertion messages are clear and easy to understand. Users can check whether their experience with the kata meets all [Kata authoring best practices](/recipes/create-kata/best-practices/). When any issues are found, users can raise issues or sugest changes in kata discourse.
```

Comment:
"Reading through description" --> "Reading through the description"

Reply:
```suggestion
Training on a kata and solving it is the first step to verify its quality, but also a very effective one. Reading through the description, users can check if it's clear enough, or something needs to be reworded or explained in a clearer way. When tests are run, reviewers can verify if they are sufficient, or whether assertion messages are clear and easy to understand. Users can check whether their experience with the kata meets all [Kata authoring best practices](/recipes/create-kata/best-practices/). When any issues are found, users can raise issues or suggest changes in kata discourse.
```